File: _posts/create_test_files.py
def create_test_md(month, date_range) :
    for i in date_range :
        with open('2014-{month:02}-{date:02}-test-page-{date:02}.md'.format(month = month, date = i), 'w+', encoding = 'utf-8') as f :
            print('---\n\
layout: post\n\
title:  "测试页面{month}-{date}"\n\
date:   "2014-{month:02}-{date:02} 20:58:01 +0800"\n\
author: kinsprite\n\
tags: [test, mytest]\n\
---\n\
\n\
测试页面{month}-{date}内容。'.format(month = month, date = i), file = f);

for m in range(1, 12) :
    create_test_md(m, range(1, 28));

File: _posts/create_test_files.rb
def create_test_md(month, date_range)
    date_range.each do |date|
        f = File.new("2014-%02i-%02i-test-page-%02i.md" % [month, date, date], 'w+:UTF-8');
        f.puts "---\n\
layout: post\n\
title:  '测试页面%02i-%02i'\n\
date:   '2014-%02i-%02i 20:58:01 +0800'\n\
author: kinsprite\n\
tags: [test, mytest]\n\
---\n\
\n\
测试页面%02i-%02i内容。" % [month, date, month, date, month, date];
    end
end

(1 .. 12).each { |m| create_test_md(m, (1 .. 28)); }
